layouts/partials/navbar-version-selector-docs.html

```diff
@@ -0,0 +1,19 @@
+{{ $jsVerSwitcher := resources.Get "js/version-switcher.js" }}
```

Comment:
This file is not in this repo. Is it missing or just somewhere else? If I remove it I can run it dev, but I do not want to make pr removing it without knowing what it does.

Reply:
This file is dynamically generated at the time of deployment. To run locally, use the sync script in the repository.

Reply:
thank you